inputfiles/overridingTypes.json

```diff
@@ -3190,5 +3202,25 @@
                 "new-type": "BodyInit"
             }
         ]
-    }
+    },
+    "namespaces": [
+        {
+            "name": "WebAssembly",
+            "methods": {
+                "method": {
+                    "instantiate": {
+                        "override-signatures": [
+                            "instantiate(bytes: BufferSource, importObject?: Imports): Promise<WebAssemblyInstantiatedSource>",
+                            "instantiate(moduleObject: Module, importObject?: Imports): Promise<Instance>"
+                        ],
+                        "force-references": [
```

Comment:
I don't think do anything?

Reply:
It does (although admittedly it's a hack). It marks `Imports` and, transitively, all its dependencies as referenced types. If you remove it, you'd need to list every single dependency manually in `knownTypes`. I believe this approach makes it a bit cleaner and better colocated with actual APIs.

Reply:
Oh wow, okay it does, clever hack 😲

Reply:
Thanks :) I was thinking about moving other `knownTypes` closer to their resepective overrides too, but eventually got lost as there's a bit too many.